dataikuapi/govern/admin_roles_permissions_editor.py

```diff
@@ -0,0 +1,433 @@
+        :rtype: :class:`dataikuapi.govern.admin_roles_permissions_editor.GovernAdminRole`
+        """
+        return GovernAdminRole(self.client, role_id)
+
+    def create_role(self, new_identifier, role_label, role_description=""):
+        """
+        Creates a new role and returns the handle to interact with it.
+
+        :param str new_identifier: Identifier of the new role
+        :param str role_label: Label of the new role
+        :param str role_description: (Optional) Description of the new role
+        :rtype: :class:`dataikuapi.govern.admin_roles_permissions_editor.GovernAdminRole`
+        """
+        result = self.client._perform_json("POST", "/admin/roles/", params={"newIdentifier": new_identifier},
+                                           body={"label": role_label, "description": role_description})
+        return GovernAdminRole(self.client, result['id'])
+
+    def list_role_assignments(self):
+        """
+        List the role assignments,each as a dict. Each dict contains at least a "blueprintId" field
+
+        :returns: A list of role assignments for each blueprint as a Python list of dict
+        :rtype: list of dict
+        """
+        return self.client._perform_json("GET", "/admin/blueprint-role-assignments")
+
+    def get_role_assignments(self, blueprint_id):
+        """
+        Get the role assignments for a specific blueprint. Returns a handle to interact with it.
+
+        :param str blueprint_id: id of the blueprint
+        :returns: A list of role assignments
+        :rtype: :class:`dataikuapi.govern.admin_roles_permissions_editor.GovernAdminBlueprintRoleAssignments`
+        """
+        assignments = self.client._perform_json("GET", "/admin/blueprint-role-assignments/%s" % (
+            blueprint_id))
+        return GovernAdminBlueprintRoleAssignments(self.client, assignments)
+
+    def create_role_assignments(self, role_assignment):
+        """
+        Create a role assignments on the Govern instance and returns the handle to interact with it.
+
+        :param: dict role_assignment: Blueprint permission as a Python dict
+        :returns: The newly created role assignment.
+        :rtype: :class:`dataikuapi.govern.admin_roles_permissions_editor.GovernAdminBlueprintRoleAssignments`
+        """
+        result = self.client._perform_json("POST", "/admin/blueprint-role-assignments", body=role_assignment)
+        return GovernAdminBlueprintRoleAssignments(self.client, result)
+
+    def get_default_permissions(self):
+        """
+        Get the default permissions. Returns a handle to interact with the permissions
+
+        :returns: A permissions object
+        :rtype: :class:`dataikuapi.govern.admin_roles_permissions_editor.GovernAdminDefaultPermissions`
+        """
+        permissions = self.client._perform_json("GET", "/admin/blueprint-permissions/default")
+        return GovernAdminDefaultPermissions(self.client, permissions)
```

Comment:
I don't think it's necessary here because the base `GovernAdminDefaultPermissions` does not hold an id so I think it will make less sense to have an object with no id and only a get_definition method